### PR TITLE
Add FosterAccount model and tests

### PR DESCRIPTION
**Pull request by ErinClaudio**
# 🔗 Issue
[#449](https://github.com/rubyforgood/pet-rescue/issues/449)

# ✍️ Description

# 📷 Screenshots/Demos

### CODE REVIEW

**Comment by ErinClaudio** [comment]
@kasugaijin @nsiwnf 
Could you please provide more details on the attributes that are expected to be included in the FosterAccount model? I noticed that the ticket mentions an 'AdopterFosterProfile' model. However, I could only locate the 'adopter_profiles' model in our current codebase. Is the 'AdopterFosterProfile' a new model that needs to be created?

thank you

**Comment by kasugaijin** [comment]
> @kasugaijin @nsiwnf Could you please provide more details on the attributes that are expected to be included in the FosterAccount model? I noticed that the ticket mentions an 'AdopterFosterProfile' model. However, I could only locate the 'adopter_profiles' model in our current codebase. Is the 'AdopterFosterProfile' a new model that needs to be created?
> 
> thank you
Hey Erin, in the ticket https://github.com/rubyforgood/pet-rescue/issues/449 it mentions that it is currently blocked by an issue that will create this resource

**Comment by ErinClaudio** [comment]
@kasugaijin Thank you, I should have read that issue ticket.

File: app/models/user.rb
[the remark this answers is not in the document]
Comment (kasugaijin) [reply app/models/user.rb:1]: yes it should have the same association as the adopter_account

**Comment by kasugaijin** [comment]
@ErinClaudio https://github.com/rubyforgood/pet-rescue/pull/478 is now merged so this is unblocked :)

**Comment by ErinClaudio** [comment]
great!!!!

**Comment by ErinClaudio** [comment]
I am not able to connect my branch to my PR. 
While my branch does reflect these changes the PR does not.
I will create a new P.

**Pull request closed without merging**